Monika-After-Story/MonikaModDev
# Small Cleanup + Fix

Moved drawing lots to its own label since dialogue for it exists in more than one place. Also mad it so the default value for drawing lots actually functions as the default instead of an alias for playing Black.

## Testing:
- Make sure drawing lots works as intended
- Make sure play again with the same rules after drawing lots will draw lots again
- Make sure when selecting drawing lots from the options, Monika doesn't say "Let's begin."
- Make sure cheat flow has no more missing labels
